config/emory/groups/admins.yml

```diff
@@ -0,0 +1,2 @@
+admin:
+  - user
```

Comment:
Could you provide more details around how this `yml` file is used?

Reply:
Good call. I intended to exclude this file from the PR but forgot to. Yes, this file contains all uids that should be granted admin status. Will remove the file from this PR because this was just for local testing. @alexBLR Can you review this after I make that change? Ayoub's out sick today.

Reply:
Removed with https://github.com/emory-libraries/dlp-lux/pull/745/commits/85d745ca448af416c652379497bf03edbbff30e5